packages/instant/src/components/scaling_amount_input.tsx

```diff
@@ -0,0 +1,53 @@
+import { BigNumber } from '@0x/utils';
+import * as _ from 'lodash';
+import * as React from 'react';
+
+import { ColorOption } from '../style/theme';
+import { util } from '../util/util';
+
+import { ScalingInput } from './scaling_input';
+
+export interface ScalingAmountInputProps {
```

Comment:
Quite a few of these are in `ScalingInputProps`, we could `pick` out the ones we care about for this interface instead of having to duplicate the type definitions.  Could go either way on this tho.

Reply:
I usually like to `extend` but the change in the `onChange` signature makes this difficult. Feel like using `pick` doesn't add a ton of value.